packages/manager/modules/pci/src/projects/project/storages/blocks/onboarding/onboarding.html

```diff
@@ -0,0 +1,12 @@
+<pci-project-empty guides="$ctrl.guides">
+    <p data-translate="pci_projects_project_storages_blocks_onboarding_content1"></p>
+    <strong data-translate="pci_projects_project_storages_blocks_onboarding_content2"></strong>
+    <p data-translate="pci_projects_project_storages_blocks_onboarding_content3"></p>
+    <p data-translate="pci_projects_project_storages_blocks_onboarding_content4"></p>
+
+    <oui-button variant="primary" text="Primary"
```

Comment:
Must be in a translation file.

Reply:
`data-`

Reply:
Nope : 
* `text` is deprecated but `ovh-ui-angular` documentation is out of date.
* we use the transclude value, so `text="Primary"` should be removed...

Reply:
So you can get rid of.